commitlint/helpers.ts

```diff
@@ -38,7 +43,7 @@ export abstract class Helpers {
     }
 
     public static assertWhen(when: string) {
```

Comment:
@realmarv you should change return type of this function to be When instead of void, in this commit (like we have done in other assert function); btw why is CI red?

Reply:
I don't know maybe some commitlint new version bug again

Reply:
> @realmarv you should change return type of this function to be When instead of void, in this commit

Why should I return When here?

Reply:
> Why should I return When here?

To do the cast, like we did from string|null to string.

> I don't know maybe some commitlint new version bug again

Then this is the highest priority thing to fix.

Reply:
It's a syntax error in cosmiconfig module. I'm not sure if it's a bug in cosmiconfig or in typescript compiler

Reply:
Well, it needs to be investigated.

Reply:
> To do the cast, like we did from string|null to string.

Let's not forget this.

Reply:
Done